db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/river/LagreJournalpostIdLoeser.kt

```diff
@@ -54,10 +94,10 @@ class LagreJournalpostIdLoeser(
                 )
                     .also { publishEvent(it) }
             } catch (ex: Exception) {
-                behov.createFail("Klarte ikke lagre journalpostId for transaksjonsId ${behov.uuid()}")
+                behov.createFail("Klarte ikke lagre journalpostId for transaksjonsId $transaksjonId")
                     .also { publishFail(it) }
-                logger.error("LagreJournalpostIdLøser klarte ikke lagre journalpostId $journalpostId for transaksjonsId ${behov.uuid()}")
-                sikkerLogger.error("LagreJournalpostIdLøser klarte ikke lagre journalpostId $journalpostId for transaksjonsId ${behov.uuid()}", ex)
+                logger.error("Klarte ikke lagre journalpost-ID $journalpostId.")
```

Comment:
bør man ikke logge med MDC her også? Evt i hvert fall ta med forespørselId / transaksjonsId for å kunne tracke flyten.